appserver/security/core-ee/src/main/java/com/sun/enterprise/security/ee/SecurityContainer.java

```diff
@@ -26,16 +26,15 @@
 
 import com.sun.enterprise.deployment.interfaces.SecurityRoleMapperFactoryMgr;
 import com.sun.enterprise.security.PolicyLoader;
-import com.sun.enterprise.security.web.integration.WebSecurityManagerFactory;
-
+import com.sun.enterprise.security.ee.web.integration.WebSecurityManagerFactory;
 import jakarta.inject.Inject;
 import jakarta.inject.Provider;
 
 /**
  * Security container service
  *
  */
-@Service(name = "com.sun.enterprise.security.ee.SecurityContainer")
+@Service(name = "com.sun.enterprise.security.ee.ee.SecurityContainer")
```

Comment:
@arjantijms, it looks like the some bad thinh - double `ee` in package name

Reply:
Thanks, fixed it!